app/src/main/java/com/example/rocketreserver/Apollo.kt

```diff
@@ -36,3 +52,89 @@ val apolloClient = ApolloClient.Builder()
     }
 
     .build()
+
+
+@Composable
+fun <D : Operation.Data> ApolloCall<D>.toState(context: CoroutineContext = EmptyCoroutineContext): State<ApolloResponse<D>?> {
```

Comment:
What's the use for `context`?

Reply:
Not really sure what the use case is to be honest - it's there in `collectAsState`, where it's used to surround `collect` with `withContext(context)`, so it felt right to have it too.